.github/workflows/main.yml

```diff
@@ -15,6 +15,101 @@ on:
 
 
 jobs:
+  windows-arm64:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        CONFIGURATION: [Debug, Release]
+        FEATURE_TOOLS: [ON, OFF]
+    env:
+      # Skip downloading necessities for the load tests since we can't test
```

Comment:
It's dependencies for the load tests and source and golden images for the other tests. Please update the comment.

Reply:
Will do